addons/base_automation/tests/test_automation.py

```diff
@@ -22,7 +22,23 @@ def test_01_on_create(self):
                 "evaluation_type": "equation",
                 "value": "False",
             })],
-        })
+        },
+        {
+            "name": "Bilbo time senstive reminder",
+            "trigger": "on_time",
+            "model_id": self.env.ref("hr_contract.model_hr_contract").id,
```

Comment:
Hello @Julien00859 @Beekan 

`hr_contract` module is not in the dependencies of `base_automation` module, and so we cannot use `hr.contract` model, we should use another model. 😬 

I don't know if you already have a fix for that. 🙂

Reply:
we are fixing it, the patch is in the forward-ports

Reply:
Thanks! @Julien00859 

Have a nice day!